src/data/staking-products.json

```diff
@@ -443,6 +443,45 @@
         "eventAction": "Clicked",
         "eventName": "Clicked Abyss Finance go to link"
       }
+    },
+    {
+      "name": "P2P.ORG",
+      "svgPath": "p2p-glyph.svg",
+      "hue": null,
+      "grayscaleLum": 0,
+      "launchDate": "2020-12-15",
```

Comment:
If users are using new smart contracts (even if audited) that have only been used since December 2022, I would use this date instead. If we're using smart contracts from genesis then the earlier date would make sense, but it doesn't sound like that's the case (please correct me if I'm wrong here).

![image](https://user-images.githubusercontent.com/54227730/225344056-a13a221c-eb92-437f-810e-6d611f66474f.png)

This is meant to indicate a level of "battle testing." If the first deposit to the smart contracts being used currently was December 2022, I would use this date to not deceive users.

```suggestion
      "launchDate": "2022-11-01",
```